tests/php/test-class-site-health.php

```diff
@@ -531,6 +533,48 @@ public function test_add_extensions() {
 		remove_filter( 'site_url', [ self::class, 'get_idn' ] );
 	}
 
+	/**
+	 * @covers \AmpProject\AmpWP\Admin\SiteHealth::is_site_has_page_cache()
+	 */
+	public function test_is_site_has_page_cache_without_page_cache() {
+
+		/**
+		 * Mock the http request.
+		 */
```

Comment:
Easy mistake to make, but this should be a PHP comment, not a docblock.

```suggestion
		/*
		 * Mock the http request.
		 */
```

Reply:
Done.